examples/next-minimal-starter/src/pages/api/trpc/[trpc].ts

```diff
@@ -13,11 +13,9 @@ const appRouter = t.router({
     // This is the input schema of your procedure
     // 💡 Tip: Try changing this and see type errors on the client straight away
     .input(
-      z
-        .object({
-          name: z.string().nullish(),
-        })
-        .nullish(),
```

Comment:
removing bullish here cause I think it's better for the demo to get the type-errors directly 🤷🏼‍♂️